base/docs/helpdb.jl

```diff
@@ -1335,11 +1335,11 @@ Get the value type of an associative collection type. Behaves similarly to `elty
 valtype
 
 doc"""
-    edit(file::AbstractString, [line])
+    edit(path::AbstractString, [line])
 
-Edit a file optionally providing a line number to edit at. Returns to the julia prompt when you quit the editor.
+Edit a file or directory optionally providing a line number to edit the file at. Returns to the julia prompt when you quit the editor.
 """
-edit(file::AbstractString, line=?)
+edit(path::AbstractString, line=?)
```

Comment:
I believe it is just indicating that the line field is optional. http://julia.readthedocs.org/en/latest/stdlib/base/#Base.edit